DOL.WHD.Section14c.Web/src/modules/components/sectionAdminWageData/adminWageDataPayTypeTemplate.html

```diff
@@ -1,5 +1,5 @@
 <answer-field answer="data.numWorkers">
-    How many workers did the employer pay {{ paytype === $parent.$parent.$parent.constants.responses.payType.hourly ? 'hourly' : 'on a piece rate basis'}}?
+    {{ paytype === $parent.$parent.$parent.constants.responses.payType.hourly ? 'How many workers with disabilities were paid an hourly subminimum wage during the most recently completed fiscal quarter?' : 'How many workers with disabilities received subminimum wages and were paid piece rates during the most recently completed fiscal quarter?' }}
```

Comment:
To be honest, I like this approach way better than the other.  It's a lot easier to read and reason about.